docs/source/concepts/ledger-model/ledger-privacy.rst

```diff
@@ -139,8 +139,8 @@ Examine each party's projection in turn:
    (again, recall that all consequences of an exercise action are a
    part of the action itself).
 
-#. Alice is a stakeholder in both the `Iou Bank A` and `PaintOffer A
-   B Bank` contracts. As all top-level actions in the ledger are
+#. Alice is a stakeholder in both the `Iou Bank A` and
+   `PaintOffer A B Bank` contracts. As all top-level actions in the ledger are
```

Comment:
Shouldn't this be double backticks? I would assume we want to render `Iou Bank A` and `PaintOffer A B` as code.

Reply:
To me it seems that on this page it's consistently in single quote. So changing it would require reworking this page.